kiota_serialization_json/json_serialization_writer.py

```diff
@@ -320,6 +340,21 @@ def write_null_value(self, key: Optional[str]) -> None:
         else:
             self.value = "null"
 
+    def write_dict_value(self, key: Optional[str], value: Dict[str, Any]) -> None:
```

Comment:
can we make this method private? we wouldn't want anyone to take a dependency on it.